ydb/public/sdk/cpp/client/ydb_query/tx.h

```diff
@@ -63,12 +63,16 @@ struct TTxSettings {
         TS_SNAPSHOT_RO
     };
 
-    const ETransactionMode Mode_;
     FLUENT_SETTING(TTxOnlineSettings, OnlineSettings);
 
+    ETransactionMode getMode() const {
+        return Mode_;
+    }
 private:
     TTxSettings(ETransactionMode mode)
         : Mode_(mode) {}
+
+    ETransactionMode Mode_;
```

Comment:
This member is not changing, better make it const

Reply:
<img width="1551" alt="image" src="https://github.com/ydb-platform/ydb/assets/169269111/6fb2de11-26cc-43bc-8b8d-e08ef88c021a">

If to return const qualifier, such errors will come out (the fact is that I use a constructor that sets Mode_, which is why it cannot be const)